pkg/skaffold/deploy/kpt.go

```diff
@@ -50,7 +50,17 @@ func (k *KptDeployer) Dependencies() ([]string, error) {
 	return nil, nil
 }
 
-func (k *KptDeployer) Cleanup(ctx context.Context, out io.Writer) error {
+func (k *KptDeployer) Cleanup(ctx context.Context, _ io.Writer) error {
+	applyDir, err := k.getApplyDir(ctx)
+	if err != nil {
+		return fmt.Errorf("getting applyDir: %w", err)
+	}
+
+	cmd := exec.CommandContext(ctx, "kpt", kptCommandArgs(applyDir, []string{"live", "destroy"}, nil, nil)...)
+	if err = util.RunCmd(cmd); err != nil {
+		return fmt.Errorf("kpt live destroy: %w", err)
+	}
```

Comment:
Can you double check that when `kpt live destroy` fails, if this can output the kpt error message? IIRC, the kpt errors would be treated as the skaffold stdout. If so, we would want the cmd to write the stdout (the real kpt error) to a buff and return the buff here.